### PR TITLE
Feature/persist settings

### PR DESCRIPTION
**Pull request by kaue-silva-ifood**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,13 +29,22 @@ npm:               ## Install node.js/npm dependencies
...
+docker-themis:        ## Run Themis in local Docker container
 	@docker run -it -p 8080:8080 \
 	-e THEMIS_DB_PASSWORD=$(THEMIS_DB_PASSWORD) \
 	-v ~/.aws/credentials:/root/.aws/credentials \
 	-e AWS_PROFILE=$(AWS_PROFILE) \
 	$(IMAGE_NAME) 
 
+docker-postgres:
+	docker run -d --rm --name faster-postgres \
+	    --network faster-net \
```
Comment (igorsmotto) [inline Makefile]: this shouldn't be here. There isn't a network in this docker composition.

File: Makefile [hunk 2]
```diff
@@ -52,7 +65,8 @@ test:              ## Run tests
 lint:              ## Run code linter to check code style
 	($(VENV_RUN); pep8 --max-line-length=120 --ignore=E128 --exclude=web,bin,$(VENV_DIR) .)
 
-server:            ## Start the server on port 8081
-	($(VENV_RUN) && eval `ssh-agent -s` && PYTHONPATH=$(dir)/src:$$PYTHONPATH bin/themis server_and_loop --port=8081 --log=themis.log)
+server:           ## Start the server on port 8081
+
```
Comment (igorsmotto) [inline Makefile]: remove empty line

File: Makefile [hunk 3]
```diff
@@ -3,6 +3,10 @@ VENV_DIR = .venv
 IMAGE_NAME = themis
 VENV_RUN = . $(VENV_DIR)/bin/activate && export PYTHONPATH=.venv/lib64/python2.7/dist-packages
 AWS_PROFILE = faster
+POSTGRES_PORT_BIND=-p 5432:5432
+THEMIS_PORTBIND=-p 8080:8080
+THEMIS_DB_URL=postgresql://faster:faster_password@localhost:5432/faster
+
```
Comment (igorsmotto) [inline Makefile:9]: remove empty line

**Review by igorsmotto — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (kaue-silva-ifood) [reply Makefile:9]: I think it would be nice a line between the variables and the first script

**Review by igorsmotto — APPROVED** [review]

**Pull request merged**